frontend/src/feature/cards/model/card.ts

```diff
@@ -0,0 +1,20 @@
+export type Card = {
+  id: string,
```

Comment:
Card list is polymorphic and contains all flashcard types, which sets of fields are different. Each card item has `type` to tell what type it is.

Reply:
yeah - and it is even easier in TS to model it, then in Java.
The Card type will change - I even left the comment in it - its WIP.